sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh

```diff
@@ -45,11 +45,15 @@
 echo Job started on:
 date -u
 
+if [[ $1 == '' ]] || [[ ! -d $1 ]]
```

Comment:
The first check here is redundant I think. If $1 is not supplied then [[ ! -d $1 ]] will return false (as '' is not a directory).